peripherals/usart/_v2_BC_common.yaml

```diff
@@ -0,0 +1,86 @@
+# Common fields between v2 B and C peripherals
+
+CR1:
+  OVER8:
+    Oversampling_16: [0, "Oversampling by 16"]
+    Oversampling_8: [1, "Oversampling by 8"]
+  EOBIE:
+    Inhibited: [0, "Interrupt is inhibited"]
+    Interrupted: [1, "A USART interrupt is generated when the EOBF flag is set in the ISR register"]
+  RTOIE:
+    Inhibited: [0, "Interrupt is inhibited"]
+    Interrupted: [1, "An USART interrupt is generated when the RTOF bit is set in the ISR register"]
+CR2:
+  RTOEN:
+    Disabled: [0, "Receiver timeout feature disabled"]
+    Enabled: [1, "Receiver timeout feature enabled"]
+  ABRMOD:
+    Start: [0, "Measurement of the start bit is used to detect the baud rate"]
+    Edge: [1, "Falling edge to falling edge measurement"]
+    Frame_7F: [2, "0x7F frame detection"]
+    Frame_55: [3, "0x55 frame detection"]
+  ABREN:
+    Disabled: [0, "Auto baud rate detection is disabled"]
+    Enabled: [1, "Auto baud rate detection is enabled"]
+  LINEN:
+    Disabled: [0, "LIN mode disabled"]
+    Enabled: [1, "LIN mode enabled"]
+  CPOL:
+    Low: [0, "Steady low value on CK pin outside transmission window"]
+    High: [1, "Steady high value on CK pin outside transmission window"]
+  CPHA:
+    First: [0, "The first clock transition is the first data capture edge"]
+    Second: [1, "The second clock transition is the first data capture edge"]
+  LBCL:
+    NotOutput: [0, "The clock pulse of the last data bit is not output to the CK pin"]
+    Output: [1, "The clock pulse of the last data bit is output to the CK pin"]
+  LBDIE:
+    Inhibited: [0, "Interrupt is inhibited"]
+    Interrupted: [1, "An interrupt is generated whenever LBDF=1 in the ISR register"]
+  LBDL:
+    Bit_10: [0, "10-bit break detection"]
+    Bit_11: [1, "11-bit break detection"]
+CR3:
+  SCARCNT: [0, 7]
+  ONEBIT:
+    Sample_3: [0, "Three sample bit method"]
+    Sample_1: [1, "One sample bit method"]
+  SCEN:
+    Disabled: [0, "Smartcard Mode disabled"]
+    Enabled: [1, "Smartcard Mode enabled"]
+  NACK:
+    Disabled: [0, "NACK transmission in case of parity error is disabled"]
+    Enabled: [1, "NACK transmission during parity error is enabled"]
+  IRLP:
+    Normal: [0, "Normal mode"]
+    LowPower: [1, "Low-power mode"]
+  IREN:
+    Disabled: [0, "IrDA disabled"]
+    Enabled: [1, "IrDA enabled"]
+BRR:
+  DIV_Mantissa: [0, 0xFFF]
+  DIV_Fraction: [0, 0xF]
```

Comment:
As per above comments on `BRR`, probably just have `BRR` here.

Reply:
Merge the fields together.